composer.json

```diff
@@ -3,18 +3,37 @@
     "description": "WordPress + Nette",
     "type": "project",
     "require": {
-        "nette/nette": "^2.4",
         "mangoweb/mangopress-sql-panel": "dev-master",
         "mangoweb/mangopress-templating": "dev-master",
         "wp-cli/wp-cli": "^1.3",
         "roots/wp-password-bcrypt": "^1.0",
-        "sentry/sentry": "^1.7"
+        "sentry/sentry": "^1.7",
+        "nette/application": "^2.4",
+        "nette/bootstrap": "^2.4",
+        "nette/caching": "^2.5",
+        "nette/component-model": "^2.4",
+        "nette/database": "^2.4",
+        "nette/finder": "^2.4",
+        "nette/forms": "^2.4",
+        "nette/http": "^2.4",
+        "nette/mail": "^2.4",
+        "nette/neon": "^2.4",
+        "nette/reflection": "^2.4",
+        "nette/robot-loader": "^3.1",
+        "nette/safe-stream": "^2.3",
+        "nette/security": "^2.4",
+        "nette/tokenizer": "^2.3",
+        "nette/utils": "^2.5",
+        "latte/latte": "^2.4",
+        "tracy/tracy": "^2.5",
+        "nette/di": "^2.4"
     },
     "config": {
       "preferred-install": "dist",
       "optimize-autoloader": true
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
```

Comment:
What's this for?

Reply:
eg MangoPressTemplating - missing properly done versioning there 🤡

Reply:
😂